src/app/pages/wallets/wallet-details/pages-wallet-details-item.model.ts

```diff
@@ -0,0 +1,47 @@
+import { TServerDateTime } from 'src/app/common/date-and-time/common-date-and-time.types';
+import { TServerEntityId } from 'src/app/common/http/common.http.types';
+import { IWalletDetailsApiResponse, IWalletDetailsTransactionApiResponse } from 'src/app/domains/wallets/domains.wallets.types';
+
+export class WalletDetailsItem {
```

Comment:
Because for now we will get wallet details by fetching all wallets and finding on the list specific one by ID for wallet you can reuse model used on wallets management page.
And only create a model for the transaction. We will treat wallet and transactions separately.